saleor/graphql/checkout/mutations.py

```diff
@@ -231,15 +230,6 @@ class CheckoutCreateInput(graphene.InputObjectType):
 
 
 class CheckoutCreate(ModelMutation, I18nMixin):
-    created = graphene.Field(
```

Comment:
This is a breaking change, I would be safer to leave this field, always return True, and add a deprecation message.